libraries-2/src/test/java/com/baeldung/handlebars/BasicUsageUnitTest.java

```diff
@@ -0,0 +1,104 @@
+package com.baeldung.handlebars;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.github.jknack.handlebars.Handlebars;
+import com.github.jknack.handlebars.Template;
+import com.github.jknack.handlebars.io.ClassPathTemplateLoader;
+import com.github.jknack.handlebars.io.TemplateLoader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+
+public class BasicUsageUnitTest {
```

Comment:
header describing what's being tested and author as well

Reply:
Fixed.